### ISSUE TITLE
Connecting PokeBuddy via Proxy

### ISSUE THREAD

**Issue by dzdam** [finndev/PokeBuddy#1024]
has anyone reached to do this? i've tried some private but it doesn't reach to connect the pokemon server.

**Comment by dragonite00123** [comment]
yep, doing this way all the time. or you can always change IP by turning off the modem around 3 to 5 mins.. and then after you connect it you have different IP

**Comment by dzdam** [comment]
@dragonite00123 which services are you using?
